pkg/cmd/dotgit/find.go

```diff
@@ -0,0 +1,99 @@
+package dotgit
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/dotgit/find"
+)
+
+func newCmdFind() *cobra.Command {
+	options := &struct {
+		treeish string
+
+		format formatType
+	}{
+		treeish: "main",
+
+		format: formatTypePlain,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "find <repository> <expression>",
+		Short: "Find files in the specified treeish",
+		Example: heredoc.Doc(`
+			$ vuls-data-update dotgit find --treeish HEAD vuls-data-raw-debian-security-tracker CVE-2025-0001.json
+			$ vuls-data-update dotgit find --treeish HEAD vuls-data-raw-debian-security-tracker CVE-2025-*.json
```

Comment:
The last arg is regexp, would `'CVE-2025-.*.json'` be better example?

Reply:
fixed: https://github.com/MaineK00n/vuls-data-update/compare/efb5092c6b3f9f211485bb8e64912ef66e5c8e04..c859dc882d35894d8c2c525162968b76174c881c#diff-64d13c34efb97181f80643cce3d7adfd915ee96e20321ef46f6fe0693ee0d858R31